modules/openshift-developer-cli-uninstalling-odo.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * assembly_creating-an-application-woth-odo.adoc  
+
+[id="uninstalling-odo"]
+
+= Uninstalling odo
+
+To uninstall `odo` you need the script `installer.sh` which is located in link:https://github.com/openshift/odo[`odo` repository on GitHub].
+
+. Run the script
+
+** If you don't have the scipt locally on your machine:
```

Comment:
* don't -> do not
* scipt -> script
* Instead of making a complicated step 1, I'd suggest to assume they don't have the script downloaded, and make that step 1. Like:

1. Download the `installer.sh` script.
2. Run the `installer.sh` script.
3. Delete local preferences...